### PR TITLE
Create ross-bio.md

### PR DESCRIPTION
**Pull request by rossathuman**
[ ] task 1
[ ] task 2

### CODE REVIEW

File: ross-bio-md [hunk 1]
```diff
@@ -0,0 +1 @@
+#About
```
Comment (sunilpaig) [inline ross-bio-md:1]: Hey! Please add your likes here!

**Comment by murthyganti** [comment]
Let me know if i need to add more details

**Comment by kaseyallen** [comment]
Only two tasks?  Nice and easy day!

**Comment by rossathuman** [comment]
One update

**Comment by rossathuman** [comment]
:+1:

**Comment by jsjain** [comment]
nice

**Pull request merged**